src/send/saga.ts

```diff
@@ -403,6 +404,11 @@ export function* sendPaymentOrInviteSaga({
     }
 
     yield put(sendPaymentOrInviteSuccess(amount))
+    if (tokenInfo?.symbol === 'CELO') {
+      ValoraAnalytics.track(CeloExchangeEvents.celo_withdraw_completed, {
```

Comment:
i have some thoughts about this event 🙈  i don't think this is in the right place/needs to exist.

This function is called every time someone makes a payment or sends an invite...seems unexpected to have a celo withdraw event every time a payment/invite is sent with celo? in any case, this information is already captured by the `send_tx_complete` event (we also send the token address with this event, we can pull analytics that way) so i am not sure that this new event is adding value.

nice to check with ellen but i think for this experiment we want to compare the withdraw flows from the "celo" screen and "add & withdraw" screen, and not include the send flow (even though it performs the same functionality, it is for a different user journey/intent)? in which case we can just check if all the interesting points of the user journey for those 2 flows are captured :)

Reply:
@kathaypacific - The send flow was included as it is likely users that would like to withdraw will use that flow. In order to accurately measure if removing `Withdraw CELO` from the exchange screen is impacting CELO withdrawals this should also be included.

Reply:
that makes sense, i don't have a problem with including the celo withdrawals in the data analysis :) but i still don't see the point of having this new event here, when the same information is available in the `send_tx_complete` event already? this event is equivalent to `send_tx_complete` when `send_tx_complete` has the token address of CELO :/ is there some reason why we need this new event?

Reply:
Moved it in https://github.com/valora-inc/wallet/pull/2488/commits/271ddcbf565095d802da27a817ad2ce0c3b034b7 to exclude invites. Having it in the `send/saga.ts` allows us to handle the entirety of data in the Firebase A/B test. Think this is expected based on our discussion.